SPECS/librdkafka/librdkafka.spec

```diff
@@ -4,8 +4,8 @@
 %define _binary_payload w9.gzdio
 Summary:        The Apache Kafka C library
 Name:           librdkafka
-Version:        1.8.2
-Release:        2%{?dist}
+Version:        2.2.0
```

Comment:
There looks to be a newer v2.3.0 which was released a couple months ago. Can you try upgrading to v2.3.0 and seeing if we can use the newer version?